### DISCUSSION TITLE
How to enable Pylint extention?

### DISCUSSION THREAD

**Discussion by Michaelzhouisnotwhite** [microsoft/vscode-pylint#27] [Q&A]
How to enable Pylint extention? The error reports always came from pylance not pylint.
![image](https://user-images.githubusercontent.com/52443370/161312963-4048c14d-6a0f-4c4a-be06-6425dee7f33b.png)

**Comment by karthiknadig** [comment]
@Michaelzhouisnotwhite It should always be enabled, the only way to disable it is using the extension view and clicking the disable extension button.

If this is not working can you look the Output > Pylint panel to see if you get any errors.

**Reply by Michaelzhouisnotwhite** [reply]
![image](https://user-images.githubusercontent.com/52443370/161314534-e0627a05-0357-47dc-aef2-38cde6491e53.png)
Thanks for reply.It seems to work in anothor conda env. My env name is  
![image](https://user-images.githubusercontent.com/52443370/161314684-a50f7781-4d66-45ea-b2d3-db751408dc1f.png)

**Reply by Michaelzhouisnotwhite** [reply]
they don't match each other

**Reply by karthiknadig** [reply]
Are you able to run `pylint` from terminal in that environment where the extension is not working?

**Reply by Michaelzhouisnotwhite** [reply]
no, I can't. But I have install it with pip. And it.work well with pylint under Python extension

**Reply by karthiknadig** [reply]
Can you set the path to the `pylint.exe` from the environment binary, and see if that works? you can set `pylint.path` setting with the pylint.exe and see.

Can you provide any info about the environment where the extension is not working so we can repro it. I recommend opening a bug.
